_posts/august-2024-newsletter.md

```diff
@@ -10,7 +10,7 @@ ogImage:
   url: "/assets/blog/august-2024-newsletter/cover.jpg"
 ---
 
-We have three upcoming events that I want you to be aware of this month.  
+DATA's August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council.
```

Comment:
Shouldn't this be in the excerpt?

Reply:
s/DATA's/Our

Reply:
The other articles have the excerpt at the top of the page as well, so I figured I'd follow the format.

Reply:
> s/DATA's/Our

Also, I figured it would be important to dictate who "we" are at the very top of the article